source/extensions/filters/http/cache/cache_headers_utils.cc

```diff
@@ -276,14 +276,14 @@ bool VaryHeader::isAllowed(const Http::ResponseHeaderMap& headers) const {
   return true;
 }
 
-bool VaryHeader::hasVary(const Http::ResponseHeaderMap& headers) {
+bool VaryUtils::hasVary(const Http::ResponseHeaderMap& headers) {
```

Comment:
This and other methods in `VaryUtils`: doesn't appear that `VaryUtils` class has any state, have you considered turning it into a namespace instead?

Reply:
I missed that the methods are all static; FWIW, I think moving them into a namespace would be a bit easier to read.